Karolis11/SEappIII
# basic asp.net/react setup (need to fix proxy still)

Installed ASP.NET Web API and React client in separate folders.
Both server and React client run separately and work, however proxying React requests to server throws Proxy Error.

To test if server works:
```
cd roommate-app
dotnet run
```
And then enter 'localhost:7030/servertest' on your browser.

To test if frontend works:
```
cd react-client
npm start
```
'localhost:3000' should contain a button 'Test server' that gives a HTTP Response Error because the request cannot yet be proxied to 'localhost:7030'.

If using Mac, could need to change 'localhost' to '127.0.0.1' in launchSettings.json (C#) and package.json (React)
